<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseDialog</class>
 <widget class="QWidget" name="BaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#BaseDialog{
background: #F5F9FF;
border:1px solid #707070;
border-radius:2px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_title" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #FFFFFF;
border-top-left-radius:2px;
border-top-right-radius:2px;
border:none;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_title_icon">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;
border:none;
/*
background-image:url(&quot;:/common_images/res/images/icons_configuration.png&quot;);
*/</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_title">
        <property name="styleSheet">
         <string notr="true">font-size: 16px;
font-weight: 500;
color: #1D2129;
line-height: 22px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>692</width>
          <height>19</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_close">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;
border:none;
border-image: url(:/icons/images/login_account_error_right.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QListView" name="list_view">
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QListView::item:selected
{
	background: #1479FF;
	color:#FFFFFF;
	border:none;
}

QListView::item
{
	height:40px;
}

QListView
{
	background:#FFFFFF;
	font-size: 14px;
	font-weight: 400;
	color: #4E5969;
	border:none;
	border-radius:0px;
}</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="itemAlignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stacked_widget">
         <property name="styleSheet">
          <string notr="true">background: #F5F9FF;
border:none;
border-radius:0px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>52</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#widget
{
	background:transparent;
	border:none;
	border-left: 1px solid #E9EEF5;
	border-top: 1px solid #E9EEF5;
	border-radius:0px;
}

QPushButton
{
	font-size: 14px;
	font-weight: 400;
	color: #FFFFFF;
	line-height: 0px;
	border-radius: 2px;
	background: #334D6F;

}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>16</number>
          </property>
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="btn_arg_config_1">
            <property name="minimumSize">
             <size>
              <width>128</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string>保存为默认值</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_arg_config_2">
            <property name="minimumSize">
             <size>
              <width>128</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string>恢复默认值</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_sys_config_1">
            <property name="minimumSize">
             <size>
              <width>96</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string>上移</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_sys_config_2">
            <property name="minimumSize">
             <size>
              <width>96</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string>下移</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>52</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_confirm">
            <property name="minimumSize">
             <size>
              <width>96</width>
              <height>32</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background: #1479FF;</string>
            </property>
            <property name="text">
             <string>生成单据</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
